_kbarticles/2024-12-10-using-opensearch-dashboards.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Understanding Opensearch Dashboards, logging and how to visualize your application logs"
+date: December 10, 2024
+excerpt: This article briefly explains what opensearch dashboards does, what types of logs are available in Dashboards, creating dashboards, and how to create log visualizations in Dashboards.
+redirect_from:
+  - /knowledge-base/2022-12-08-common-kibana-searches-on-clouddotgov/
+
+---
+
+## What does Dashboard do?
+
+Dashboard is a user interface that lets you search and visualize your application logs. Dashboard has a [user guide](https://opensearch.org/docs/latest/dashboards/) that explains more about how to use it and to create custom visualizations.
+
+## What are some types of logs?
+
+Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type. HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).
+
+Cloudfoundry logs are often translated into field names in Opensearch/Dashboards using the log type as a prefix. For example, APP logs in Dashboards include fields like `app.name` for the application name.
```

Comment:
This isn't true. I think it's referring to how JSON logs were parsed into Elasticsearch.

If a JSON log comes in now and contains a key of `name`, then you could search `app.name`, but it all works very differently and there is no creation of custom fields